views/meta-box-subscription-info.php

```diff
@@ -260,57 +260,196 @@
 	</tr>
 
 	<tr>
-		<th scope="row">
-			<?php esc_html_e( 'Cancel URL', 'pronamic_ideal' ); ?>
-		</th>
-		<td>
+		<th scope="row" rowspan="3">
+			<?php esc_html_e( 'Action links', 'pronamic_ideal' ); ?>
+
+			<span class="dashicons dashicons-editor-help pronamic-pay-tip" title="<?php echo esc_attr__( 'These actions links can be shared with the customer.', 'pronamic_ideal' ); ?>" tabindex="0"></span>
+
 			<?php
 
-			$url = $subscription->get_cancel_url();
+			/**
+			 * Clipboard feature.
+			 * 
+			 * @link https://github.com/WordPress/WordPress/blob/68e3310c024d7fceb84a5028e955ad163de6bd45/wp-includes/js/plupload/handlers.js#L364-L393
+			 * @link https://translate.wordpress.org/projects/wp/dev/nl/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=10763746&filters%5Btranslation_id%5D=91929960
+			 * @link https://translate.wordpress.org/projects/wp/dev/nl/default/?filters%5Bstatus%5D=either&filters%5Boriginal_id%5D=6831324&filters%5Btranslation_id%5D=58732256
+			 */
 
-			printf(
-				'<a href="%s">%s</a>',
-				esc_attr( $url ),
-				esc_html( $url )
-			);
+			wp_enqueue_script( 'clipboard' );
 
 			?>
+			<script>
+				function pronamic_pay_action_links() {
+					var clipboard = new ClipboardJS( '.pronamic-pay-copy-url' ),
+						successTimeout;
+
+					clipboard.on( 'success', function( event ) {
+						var triggerElement = jQuery( event.trigger ),
+							successElement = jQuery( '.success', triggerElement.closest( '.pronamic-pay-action-link-clipboard' ) );
+
+						event.clearSelection();
+
+						clearTimeout( successTimeout );
+
+						successElement.removeClass( 'hidden' );
+
+						successTimeout = setTimeout( function() {
+							successElement.addClass( 'hidden' );
+						}, 3000 );
+					} );
+				}
+
+				addEventListener( 'DOMContentLoaded', function ( event ) {
+					pronamic_pay_action_links();
+				} );
+			</script>
+
+			<style>
+				.pronamic-pay-action-link {
+					display: flex;
+
+					flex-direction: column;
+
+					gap: 5px;
+				}
```

Comment:
Is this needed? Doesn't seem to have much effect on the styling, except for some random `gap` spacing?

Reply:
Random `gap` spacing? It's a fixed `gap` of `5px`?  It can also be done with margin/padding, I thought this was flex 😏

Reply:
Sorry, with ‘random gap’ I meant to say that a regular paragraph also adds some spacing if needed?